src/facade.cpp

```diff
@@ -27,6 +27,10 @@ mrb_value docopt_value_to_mrb_value(const docopt::value& value, mrb_state *mrb)
 
 extern "C" mrb_value parse(char* usage, int argc, const char** argv, mrb_state *mrb) {
     mrb_value options = mrb_hash_new(mrb);
+
+    if (argc <= 0)
```

Comment:
I'm used to put one line block, just after the condition, especially if it fits into 80cols

`if (argc <= 0) return options;`